README.md

```diff
@@ -1,7 +1,7 @@
 clj-multihash
 =============
 
-[![Dependency Status](https://www.versioneye.com/clojure/mvxcvi:multihash/badge.svg)](https://www.versioneye.com/clojure/mvxcvi:multihash)
+[![Dependency Status](https://www.versioneye.com/clojure/mvxcvi:multihash/2.0.1/badge.svg)](https://www.versioneye.com/clojure/mvxcvi:multihash)
```

Comment:
I actually don't think we should correct this one. Otherwise this will need to be updated every time a new version is released, and experience shows that it inevitably gets forgotten in the release process.